### PR TITLE
changed the font size in for issue 2598 problem no 5

### PR DESCRIPTION
**Pull request by taher-mulla**
I have worked on Issue #2598 and have changed the font size in the box for it to 20px.

### CODE REVIEW

File: bin/themes/Dark/style.css [hunk 1]
```diff
@@ -3,70 +3,70 @@
 /* Use 'url(./xxxx)' for resources, it will be replaced with the right path at runtime */
 
 ShortcutEdit[error="true"] {
-    color: #EF5350;
```
Comment (mrexodia) [inline bin/themes/Dark/style.css:6]: With all these whitespace changes it's not possible to see what you actually changed. Please configure your editor to not reformat the whole file.

File: bin/themes/Dark/style.css [hunk 2]
```diff
@@ -89,502 +89,500 @@ QCheckBox::indicator:checked:pressed,
...
-    border: 1px solid #212121;
-    border-radius: 2px;
-    color: #e0e0e0;
+  background-color: #313131;
+  padding: 1px;
+  border-style: solid;
+  border: 1px solid #212121;
+  border-radius: 2px;
+  color: #e0e0e0;
+  font-size: 20px;
```
Comment (mrexodia) [inline bin/themes/Dark/style.css:303]: I'm not sure if you should change all the `QLineEdit`. Perhaps instead set the font size in the calculator directly...

**Review by mrexodia — CHANGES_REQUESTED** [review]

**Comment by taher-mulla** [comment]
I have changed the .css file. I have also added the font size to 4. The default should be 3 according to what I see online.

**Comment by mrexodia** [comment]
Thank you, but this is not fixing the issue because font size 4 looks super small on my screen. Likely the fix would involve using the 'default+x%' font size based on the user's CSS but it is too minor for me to spend a lot of time on.

I'll close this for now, feel free to create a new PR.

**Pull request closed without merging**